numba_dpex/dpnp_iface/arrayobj.py

```diff
@@ -158,24 +159,29 @@ def build_dpnp_ndarray(
             filter selector string, an instance of :class:`dpctl.SyclDevice`
             corresponding to a non-partitioned SYCL device, an instance of
             :class:`dpctl.SyclQueue`, or a `Device` object returnedby
-            `dpctl.tensor.usm_array.device`. Default: `"unknwon"`.
-        queue (:class:`dpctl.SyclQueue`, optional): Not supported.
-            Default: `None`.
+            `dpctl.tensor.usm_array.device`. Default: `None`.
+        sycl_queue (:class:`dpctl.SyclQueue`, optional): The SYCL queue to use
+            for output array allocation and copying. sycl_queue and device
+            are exclusive keywords, i.e. use one or another. If both are
+            specified, a TypeError is raised unless both imply the same
```

Comment:
The last portion of the docstring is no longer valid. Both queue and device cannot be specified anymore.

Reply:
The same review comment applies to all other docstrings that have same text.